micro-rdk/src/common/app_client.rs

```diff
@@ -255,19 +255,18 @@ impl AppClient {
     // `last_reconfigured` values for resource statuses.
     pub async fn get_config(
         &self,
-        ip: Ipv4Addr,
+        ip: Option<Ipv4Addr>,
     ) -> Result<(Box<ConfigResponse>, Option<DateTime<FixedOffset>>), AppClientError> {
-        let agent = AgentInfo {
+        let agent = ip.map(|ip| AgentInfo {
```

Comment:
before this is merged we should make sure the absence of AgentInfo as no effect on what is displayed on app.viam.com

Reply:
From my testing, it doesn't change what is displayed.